src/box/schema_def.c

```diff
@@ -86,3 +86,11 @@ schema_object_name(enum schema_object_type type)
 	assert((int) type < (int) schema_object_type_MAX);
 	return object_type_strs[type];
 }
+
+const char *
+schema_entity_name(enum schema_object_type type)
+{
+	assert((int) type > (int) schema_object_type_MAX);
+	assert((int) type < (int) schema_entity_type_MAX);
+	return object_type_strs[(type % (schema_object_type_MAX)) + 1];
```

Comment:
Please make it consistent with `schema_entity_type` function. I's quite misleading that we convert normal object ID to entity ID via switch and back via arithmetic.
BTW I like arithmetic more.
I would make a couple of functions to convert back and forth and use one of them here.

Reply:
Added a couple of helpers in a separate patch. See on the branch